manifest.json

```
@@ -29,7 +28,7 @@
         }
     ],
     "requirements": {
-        "yunohost": ">= 4.2.0"
+        "yunohost": ">= 4.3.0"
```

```suggestion
        "yunohost": ">= 4.3.1.8"
```

because of using YNH_ARCH